modules/appeals_api/app/controllers/appeals_api/v0/appeals_controller.rb

```diff
@@ -20,6 +20,16 @@ def index
         )
       end
 
+      def show_higher_level_review
+        higher_level_review = review_service.get_higher_level_reviews(params[:uuid])
```

Comment:
This is an anti-pattern in conventional Rails controller. We should consider opening a separate controller that maps more appropriately to this resource - i.e. `/v0/higher_level_review` 

Perhaps y'all have an argument otherwise - but I don't see why this can't be it's own resource